scripts/test/run-integration-tests.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+set -e
+OS_OVERRIDE=$(go env GOOS)
+pushd ./test/integration-new
+
+echo "Running integration test with the following configuration:
+KUBE CONFIG: $KUBECONFIG
+CLUSTER_NAME: $CLUSTER_NAME
+AWS_REGION: $AWS_REGION
+OS_OVERRIDE: $OS_OVERRIDE"
+
+while getopts "f:" arg; do
+  case $arg in
+    f) FOCUS=$OPTARG;
+    echo "FOCUS: $FOCUS";
+  esac
+done
+    
+
+if [[ -z "${OS_OVERRIDE}" ]]; then
+    OS_OVERRIDE=linux
+fi
+
+CLUSTER_INFO=$(aws eks describe-cluster --name $CLUSTER_NAME --region $AWS_REGION)
+
+VPC_ID=$(echo $CLUSTER_INFO | jq -r '.cluster.resourcesVpcConfig.vpcId')
+SERVICE_ROLE_ARN=$(echo $CLUSTER_INFO | jq -r '.cluster.roleArn')
+ROLE_NAME=${SERVICE_ROLE_ARN##*/}
+ 
+
+
+START=$SECONDS
+
+for dir in */  
+do
+
+    cd "${dir%*/}"  
+    if [ -n "$FOCUS" ]; then
+        ginkgo --focus="$FOCUS" -v -r -- --cluster-kubeconfig=$KUBECONFIG --cluster-name=$CLUSTER_NAME --aws-region=$AWS_REGION --aws-vpc-id=$VPC_ID
```

Comment:
with `set -e` if the firs test fails then all the other tests will be skipped as the script will exit immediately. It would be nice if we can allow all suites to run and later return with non zero exit code if any one suite failed. 

Did something similar here - https://github.com/aws/amazon-vpc-resource-controller-k8s/blob/master/scripts/test/test-with-eksctl.sh#L136-L142